.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+LineLength:
+  Enabled: false
```

Comment:
line length issues are usually easy to fix, or just extend the length a little.  personal preference though so up to you.